.gitignore

```diff
@@ -7,3 +7,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+*.DS_Store
```

Comment:
This can stay, but protip: create a global gigignore `git config --global core.excludesfile ~/.gitignore`